templates/osbs-buildroot-imagestream.yml.j2

```diff
@@ -3,7 +3,7 @@ kind: ImageStream
 metadata:
   annotations:
     openshift.io/image.insecureRepository: {{ osbs_insecure_repository }}
-  name: osbs-buildroot
+  name: osbs-rcm-buildroot
```

Comment:
Com to think of it, this should probably be a variable.

Reply:
ok